src/skins/vector/views/pages/Login.js

```diff
@@ -0,0 +1,199 @@
+/*
```

Comment:
The intention is for this to be a complete component. Based on your comments, would you prefer if it were in the components directory to avoid confusion?